### PR TITLE
[15.0][IMP] sale_elaboration: Based report view elaborations in stock moves instead of sale.order.lines

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT51619

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

File: sale_elaboration/models/sale_order.py [hunk 1]
```diff
@@ -53,11 +52,6 @@ class SaleOrderLine(models.Model):
     elaboration_price_unit = fields.Float(
         "Elab. Price", compute="_compute_elaboration_price_unit", store=True
     )
-    is_prepared = fields.Boolean(
-        compute=lambda self: None,
-        search="_search_is_prepared",
-        help=("Dummy field to be able to find prepared lines"),
-    )
```
Comment (CarlosRoca13) [inline sale_elaboration/models/sale_order.py:60]: Why is this beeing removed?

**Comment by sergio-teruel** [comment]
@CarlosRoca13 With this new approach the records of elaboration wiew are stock moves so is not needed have a fiel in sale order line to get is a sol is prepared or no

**Comment by CarlosRoca13** [comment]
Ok! Thanks for the clarification :smile:

**Review by carlosdauden — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**